### PR TITLE
Feature/issue 26 sample stream

### PR DESCRIPTION
**Pull request by g8rswimmer**
This handles the tweet streaming.

@dnapier here is the PR for the sample stream, will be working on the filtered version soon.

### CODE REVIEW

File: v2/_examples/tweet-sample-stream/main.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+				fmt.Println("closing")
+				return
+			case tm := <-tweetStream.Tweets():
+				tmb, err := json.Marshal(tm)
+				if err != nil {
+					fmt.Printf("error decoding tweet message %v", err)
+				}
+				outputFile.WriteString(fmt.Sprintf("tweet: %s\n\n", string(tmb)))
+				outputFile.Sync()
+				fmt.Println("tweet")
```
Comment (Guitarbum722) [inline v2/_examples/tweet-sample-stream/main.go:69]: This is funny...don't know why.

File: v2/tweet_stream.go [hunk 2]
```diff
@@ -23,14 +27,54 @@ const (
 	// ErrorMessageType is the error system message type
 	ErrorMessageType SystemMessageType = "error"
 
-	tweetStart = "data"
+	tweetStart  = "data"
+	keepAliveTO = 11 * time.Second
```
Comment (Guitarbum722) [inline v2/tweet_stream.go:31]: I'm not totally clear what this is set to 11 seconds for...let me know if I'm missing something obvious though.

Comment (Guitarbum722) [reply v2/tweet_stream.go:31]: I'm assuming you are allotting time to wait for a heartbeat at 10 seconds as described in their docs...just confirming at this point.

**Review by Guitarbum722 — APPROVED** [review]
One comment not necessarily blocking.  Sweet!

File: v2/tweet_stream.go (hunk 2 above)
Comment (g8rswimmer) [reply v2/tweet_stream.go:31]: Yes, it was based off of the documentation of a 10 second heartbeat and I was giving a second buffer on that.

File: v2/_examples/tweet-sample-stream/main.go (hunk 1 above)
Comment (g8rswimmer) [reply v2/_examples/tweet-sample-stream/main.go:69]: It is actually pretty cool to run the example and see it work

**Pull request merged**